### ISSUE TITLE
Server crash on restart / Connection Timed out

### ISSUE THREAD

**Issue by WillisShek** [AllTheMods/ATM-5#39]
Log: [https://pastebin.com/rwyAyc1G](url)

It mentions that the tick was taking way too long. The crash happened when the server restarted by not running. I wonder whether it is caused by chunking loading from the anchor upgrade of Mekanism. I will try to disable watchdog and see whether it helps.

**Comment by WillisShek** [comment]
The server won't crash now after updating to 1.10 and disabling watchdog. However, no one can log in as the server always return timed out even if readTimeout is set to 900.

**Comment by rezonant** [comment]
I recommend posting the full log- we've experienced some endless tick issues that we've solved in various ways, sometimes you'll get an exception printed that does not cause a full crash before Watchdog kills the server due to the endless tick. The crash report won't give you much useful information in the case of an endless tick issue.

Disabling watchdog is only going to stop the server from crashing, the tick is still likely to be stuck. The default watchdog interval is 60 seconds -- if that's not enough for a single tick, probably no amount of time will be enough.
